eeg_cofitage/code/COFITAGE_base.py

```diff
@@ -0,0 +1,102 @@
+import logging
```

Comment:
What is this script about? It would be nice to have "docstrings", as in the python convention described in [pep257](https://www.python.org/dev/peps/pep-0257/)

1. A module docstring (see [this for help](https://stackoverflow.com/questions/2557110/what-to-put-in-a-python-module-docstring))
1. docstrings for each function

Reply:
- I was unable to put the events into the `_eeg.edf` file, so I put them into a separate file `_events.edf`. This is not strictly forbidden in edf+ standard, they even put an example of annotation-only edf file.
But I agree its not following BIDS standard and I considering to keep events only in tsv file
- The `COFITAGE_base.py` is a plug-in file that do a experiment-specific treatment in more general conversion and bidsifier tool. Essentially it defines modalities and do basic sanity checks. I will add doc strings.

Reply:
Very interesting first point, I did not know about that. If you feel like it would be good for BIDS to accept `_events.edf` files, you could open an issue on the main repository and argue for your point. 

It would make the EDF format a "multi file format", like EEGLAB (.set, .fdt) or BrainVision (.vhdr, .eeg, .vmrk) --> but if there are benefits to it, the community might agree to accept it.

Reply:
Actually I would prefer lift the restrictions on data formats and data files all together. I will make a suggestion in issues on bids.

The separate edf file for events is a hack -- not all edf reader supports it (edfbrowser do, mne don't). So I will probably remove _events.edf.

Reply:
- Added docstrings to `COFITAGE_base.py`
- Changed output format to vhdr, headers uncut, datafiles are empty

Reply:
The docstrings do not follow the convention in [pep257](https://www.python.org/dev/peps/pep-0257/), but it's already an improvement, thanks!